eth/backend.go

```diff
@@ -323,6 +323,13 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 	if err != nil {
 		return nil, err
 	}
+	if config.PruneAncientData && config.BlockHistory != 0 {
+		log.Warn("The node enables PruneAncientData, BlockHistory wille force to 0")
+		config.BlockHistory = 0
+	}
+	if err = eth.blockchain.PruneBlockHistory(config.BlockHistory); err != nil {
```

Comment:
run PruneBlockHistory when start nodes?
the prune is not revertable, so I think make user to use it as a tool is better.
when they use this tool, they make their purpose unambiguously

Reply:
The node runs as normal in default config(BlockHistory=0), they must set BlockHistory>0 that will trigger the pruning.